docker/gcp_pudl_etl.sh

```diff
@@ -186,20 +186,22 @@ function notify_slack() {
     upload_file_to_slack "$LOGFILE" "$BUILD_ID logs:"
 }
 
-function update_nightly_branch() {
+function merge_tag_into_branch() {
```

Comment:
I think we're doing exactly the same thing in these two cases, just with a different tag and a different persistent branch that it needs to be merged into, so having a single function seems ideal. Also it'll get tested in the nightly builds rather than only when we run a release which is nice.